portal-web/docroot/html/common/themes/top_messages.jsp

```diff
@@ -61,24 +50,26 @@ if (Validator.isNotNull(jspPath) || Validator.isNotNull(message)) {
 		</div>
 	</div>
 
-	<aui:script use="liferay-notice">
-		var banner = new Liferay.Notice(
-			{
-				animationConfig:
-					{
-						duration: 2,
-						top: '0px'
-					},
-				closeText: false,
-				node: '#portalMessageContainer',
-				noticeClass: 'hide <%= cssClass %>',
-				timeout: <%= timeout %>,
-				toggleText: false,
-				useAnimation: <%= useAnimation %>
+	<aui:script>
```

Comment:
Should I use a `<script>` tag instead of `aui:script` taglib?

Reply:
If there is no dependency on AUI, there is no benefit to using `aui:script`, AFAIK. At least, that was my conclusion when I looked at the difference in the generated output; eg. in [this commit](https://github.com/brianchandotcom/liferay-portal/pull/88591/commits/44eb433d6a2c44c40c71fa24ba919d1db6dd33ad), I wrote:

> In terms of generated output, the only difference is that the `aui:script` tag makes an IIFE that sets up some local variables that the scripts don't use:
>
> ```
> var $ = AUI.$;var _ = AUI._;
> ```

So, that's at least how I understand things work.